### PR TITLE
SNO-144-redirect-downloads-through-nginx-or-apache

### PR DESCRIPTION
**Pull request by lrowe**
This is the encoded change to accompany encode-dcc/snovault#192.

### CODE REVIEW

File: ini-templates/production-template.ini [hunk 1]
```diff
@@ -5,6 +5,7 @@ accession_factory = ${accession_factory}
 blob_bucket = ${blob_bucket}
 blob_store_profile_name = ${file_upload_profile_name}
 file_upload_bucket = ${file_upload_bucket}
+accel_redirect_header = ${accel_redirect_header}
```
Comment (keenangraham) [inline ini-templates/production-template.ini]: Probably want this in base.ini so it applies to demos as well. @caseylitton can confirm.

**Review by keenangraham — CHANGES_REQUESTED** [review]
We'll have to point this to the sno branch so tests run and we can spin up demo.

**Comment by keenangraham** [comment]
Putting param in base.ini and pointing to sno branch.

**Comment by keenangraham** [comment]
Rebased.

File: ini-templates/production-template.ini (hunk 1 above)
Comment (lrowe) [reply ini-templates/production-template.ini]: I thought demo instances ran production.ini? Removing the necessity to run nginx in development seems useful.

Comment (keenangraham) [reply ini-templates/production-template.ini]: Yeah, I think you are right. Concerned the `if proxy and accel_redirect_header` logic would change current behavior (local and demo) if the `accel_redirect_header` param wasn't available to both INIs.

Comment (keenangraham) [reply ini-templates/production-template.ini]: If that makes sense to you should we make symmetric change to snovault PR?

**Review by keenangraham — APPROVED** [review]

**Pull request merged**